lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -2055,6 +2067,10 @@ namespace Js
         {
             func(this->deferredPrototypeType);
         }
+        if (this->undeferredFunctionType)
```

Comment:
Since this is a path type, we should be able to enumerate those by walking that tree, instead of registering it to the functionObjectTypeList.

Reply:
This change isn't making the undeferredFunctionType into a path type. It's still a simple type. In any case, it will get registered as a function object type when we create it, so we _probably_ don't need these lines.